extensions/map-layers/package.json

```diff
@@ -36,24 +36,23 @@
     "url": "http://www.bentley.com"
   },
   "devDependencies": {
-    "@itwin/core-bentley": "workspace:*",
+    "@bentley/react-scripts": "4.0.3",
```

Comment:
there should be no need for map-layers to pull this pkg in; i thought i had removed it previously

Reply:
Should be able to remove cra for the tsconfig types safely: extensions/map-layers/tsconfig.json
If it complains about the svg?sprite syntax we just need to add a *.d.ts here for that

Reply:
My bad, looks like i only made this change in 2.x: https://github.com/iTwin/itwinjs-core/pull/2518, I couldve sworn i removed it on master

Reply:
made the update, resolving.